build.gradle

```diff
@@ -66,8 +66,13 @@ allprojects {
         }
     }
 
+    task updateInstallerVersion() {
+        installerVersion = UUID.randomUUID().toString()
```

Comment:
@parkavi11 This value should be updated in the  gradle.properties file. 

https://github.com/ballerina-platform/ballerina-distribution/blob/master/gradle.properties#L70